lib/uplinkc/testdata/access_scope_test.c

```diff
@@ -27,6 +27,14 @@ int main(int argc, char *argv[])
 
         free_scope(scope);
     }
+    { //restrict_scope - checks scope
+	ScopeRef scope; //just need a valid scope-variable for this test 
```

Comment:
you can use code from line 18 https://github.com/storj/storj/pull/3705/files#diff-9d4a5b25546454e43804717df7b8b38dL18

Reply:
Not in this case - here I want to check wether I get an error if the scope is invalid. And it should be invalid if I pass an empty variable/pointer/reference.

Reply:
Don't know how deep the tests should go, though.